src/applications/ivc-champva/10-7959C/chapters/certifierInformation.js

```diff
@@ -120,22 +112,18 @@ export const certifierPhoneEmail = {
 
 export const certifierRelationship = {
   uiSchema: {
-    ...titleUI(
-      'Your relationship to the applicant',
-      `You can add up to ${MAX_APPLICANTS} applicants on a single application. If you need to add more than ${MAX_APPLICANTS}, you’ll need to fill out another form for them.`,
-    ),
+    ...titleUI('Your information'),
     certifierRelationship: {
-      relationshipToApplicants: checkboxGroupUI({
+      relationshipToApplicants: radioUI({
```

Comment:
Switched from checkbox to radio UI since there will now only be one applicant with whom the certifier has a relationship.